diff_drive_controller/package.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0"?>
+<package format="3">
+  <name>diff_drive_controller</name>
+  <version>0.0.1</version>
+  <description>Controller for executing joint-space trajectories on a group of joints</description>
+  <maintainer email="allabana@amazon.com">Anas Abou Allaban</maintainer>
+
+  <license>Apache License 2.0</license>
+
+  <buildtool_depend>ament_cmake</buildtool_depend>
+
+  <build_depend>controller_interface</build_depend>
+  <build_depend>hardware_interface</build_depend>
+  <build_depend>pluginlib</build_depend>
+  <build_depend>rclcpp</build_depend>
+  <build_depend>rclcpp_lifecycle</build_depend>
+  <build_depend>rcutils</build_depend>
+  <build_depend>realtime_tools</build_depend>
+  <build_depend>geometry_msgs</build_depend>
+  <build_depend>nav_msgs</build_depend>
+  <build_depend>tf2_msgs</build_depend>
+  <build_depend>libboost-dev</build_depend>
```

Comment:
This will still drag in just about everything of Boost, as this resolves to `libboost1.65-dev` on (fi) Ubuntu Bionic, which depends on [these additional packages](https://packages.ubuntu.com/bionic/libboost1.65-dev).

Reply:
okay, so how do I specify to only pull in the math library? I don't think a deb package exists to only pull the math library in without pulling everything else in.

Reply:
Ah, I believe I see now where this comes from:

```
$ dpkg -S boost/accumulators/accumulators.hpp
libboost1.65-dev:amd64: /usr/include/boost/accumulators/accumulators.hpp
```

Reply:
I believe `libboost-math` is only a library which ships the runtime library.

The headers are still part of `libboost-dev`. So this key would seem to be correct.

As a run dependency it would have been nice to be able to spec [libboost-math](https://packages.ubuntu.com/bionic/libboost-math1.65.1), but that key doesn't exist, as you note.

Reply:
I see so the runtime components of math are in libboost-math but all header only math components live with the core library.